bin/anthology/__init__.py

```diff
@@ -3,3 +3,11 @@
 from .people import PersonName
 from .papers import Paper
 from .volumes import Volume
+
+import warnings
+
+warnings.warn(
+    """Your code uses the legacy Anthology library.  Consider migrating to acl-anthology-py <https://acl-anthology-py.readthedocs.io/>""",
+    FutureWarning,
+    stacklevel=2,
+)
```

Comment:
Great idea here. Should we add a target deprecation date? 2025Q1?

Reply:
That seems ambitious :) and I don't know if we really _need_ one. It depends on our availability and time to update scripts, after all.